<launch>
  <include file="$(find crazy_land)/launch/vicon.launch" />

  <node pkg="crazy_land" type="jackal_control" name="jackal_control" output="screen"/>

  <node pkg="crazy_land" type="jackal_trajectory" name="jackal_trajectory" output="screen">
  </node>
</launch>
